interop-cats/jvm/src/main/scala/scalaz/zio/interop/catsjvm.scala

```diff
@@ -30,53 +30,72 @@ abstract class CatsPlatform extends CatsInstances {
   }
 
-  implicit def ioTimer[R <: Clock, E]: effect.Timer[ZIO[R, E, ?]] = new effect.Timer[ZIO[R, E, ?]] {
-    override def clock: cats.effect.Clock[ZIO[R, E, ?]] = new effect.Clock[ZIO[R, E, ?]] {
-      override def monotonic(unit: TimeUnit): ZIO[R, E, Long] =
+  implicit def ioTimer[R <: Clock]: effect.Timer[TaskR[R, ?]] = new effect.Timer[TaskR[R, ?]] {
+    override def clock: cats.effect.Clock[TaskR[R, ?]] = new effect.Clock[TaskR[R, ?]] {
+      override def monotonic(unit: TimeUnit): TaskR[R, Long] =
         zioClock.nanoTime.map(unit.convert(_, NANOSECONDS))
 
-      override def realTime(unit: TimeUnit): ZIO[R, E, Long] =
+      override def realTime(unit: TimeUnit): TaskR[R, Long] =
         zioClock.currentTime(unit)
     }
 
-    override def sleep(duration: FiniteDuration): ZIO[R, E, Unit] =
+    override def sleep(duration: FiniteDuration): TaskR[R, Unit] =
       zioClock.sleep(scalaz.zio.duration.Duration.fromNanos(duration.toNanos))
   }
 
-  implicit val taskEffectInstances: effect.ConcurrentEffect[Task] with SemigroupK[Task] =
-    new CatsConcurrentEffect
+  implicit def taskEffectInstances(implicit runtime: Runtime[Any]): effect.ConcurrentEffect[Task] =
+    new CatsConcurrentEffect { override val rts = runtime }
 
-  implicit val taskParallelInstance: Parallel[Task, Util.Par] =
+  implicit def taskConcurrentInstances[R]: effect.Concurrent[TaskR[R, ?]] =
+    new CatsConcurrent[R]
+
+  implicit def taskParallelInstance(implicit rts: Runtime[Any]): Parallel[Task, Util.Par] =
     parallelInstance(taskEffectInstances)
 }
 
 sealed abstract class CatsInstances1 extends CatsInstances2 {
-  implicit def ioMonoidInstances[E: Monoid]: MonadError[IO[E, ?], E] with Bifunctor[IO] with Alternative[IO[E, ?]] =
-    new CatsAlternative[E] with CatsBifunctor
+  implicit def ioMonoidInstances[R, E: Monoid]
+    : MonadError[ZIO[R, E, ?], E] with Bifunctor[ZIO[R, ?, ?]] with Alternative[ZIO[R, E, ?]] =
+    new CatsAlternative[R, E] with CatsBifunctor[R]
 
-  implicit def parallelInstance[E](implicit M: Monad[IO[E, ?]]): Parallel[IO[E, ?], ParIO[E, ?]] =
-    new CatsParallel[E](M)
+  implicit def parallelInstance[R, E](implicit M: Monad[ZIO[R, E, ?]]): Parallel[ZIO[R, E, ?], ParIO[R, E, ?]] =
+    new CatsParallel[R, E](M)
 }
 
 sealed abstract class CatsInstances2 {
-  implicit def ioInstances[E]: MonadError[IO[E, ?], E] with Bifunctor[IO] with SemigroupK[IO[E, ?]] =
-    new CatsMonadError[E] with CatsSemigroupK[E] with CatsBifunctor
+  implicit def ioInstances[R]
+    : MonadError[TaskR[R, ?], Throwable] with Bifunctor[ZIO[R, ?, ?]] with SemigroupK[TaskR[R, ?]] =
+    new CatsMonadError[R, Throwable] with CatsSemigroupK[R, Throwable] with CatsBifunctor[R]
 }
 
-private class CatsConcurrentEffect extends CatsConcurrent with effect.ConcurrentEffect[Task] {
+private abstract class CatsConcurrentEffect
+    extends CatsConcurrent[Any]
+    with effect.ConcurrentEffect[Task]
+    with effect.Effect[Task] {
+  def rts: Runtime[Any]
```

Comment:
Change this to a constructor parameter: `Runtime[R]`, so you can `provide` the `R` before running or converting to Cats Effect `IO`.